scripts/pg/master.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+mkdir -p $HOME/logs/pg/master
+exec &>> $HOME/logs/pg/master/run.log
+
+echo "-----------$(date)"
+
+API_NAME=$1
+API_PASSWORD=$2
+PATTERNS=$3
+
+set -x
+
+docker pull rubybench/pg_master
+
+docker run --name postgres -d postgres:9.3.5
```

Comment:
In that case I must change version in sequel and rails scripts to have consistent environment.